1_crosswalk_fetch.yml

```diff
@@ -127,6 +128,16 @@ targets:
     command: slice_iadnr_contour(target_name,
       contour_ind = '1_crosswalk_fetch/out/iadnr_contours_sf.rds.ind')
 
+  1_crosswalk_fetch/out/isro_contours_sf.rds.ind:
+    command: fetch_isro_lakes(target_name,
+      use_geoms = I('Multi Polygon'),
+      gdb_filename = I('ISRO_GLKN_BathymetricData.gdb'),
+      zip_ind = '3_params_fetch/in/ISRO_GLKN_Bathymetric_FinalData.zip.ind')
+
+  1_crosswalk_fetch/out/isro_lakes_sf.rds.ind:
```

Comment:
Should this get added under the overall makefile depends list for `1_crosswalk_fetch`?